mmv1/products/gkeonprem/VmwareAdminCluster.yaml

```
@@ -0,0 +1,706 @@
+    output: true
+  - type: Boolean
+    name: "reconciling"
+    description: |
+      If set, there are currently changes in flight to the VMware admin cluster.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: "createTime"
+    description: |
+      The time the cluster was created, in RFC3339 text format.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: "updateTime"
+    description: |
+      The time the cluster was last updated, in RFC3339 text format.
+    output: true
+  - type: String
+    name: "localName"
+    description: |
+      The object name of the VMwareAdminCluster custom resource on the
+      associated admin cluster. This field is used to support conflicting
+      names when enrolling existing clusters to the API. When used as a part of
+      cluster enrollment, this field will differ from the ID in the resource
+      name. For new clusters, this field will match the user provided cluster ID
+      and be visible in the last component of the resource name. It is not
+      modifiable.
+      All users should use this name to access their cluster using gkectl or
+      kubectl and should expect to see the local name when viewing admin
+      cluster controller logs.
+    output: true
+  - type: String
+    name: "etag"
+    description: |
+      This checksum is computed by the server based on the value of other
+      fields, and may be sent on update and delete requests to ensure the
+      client has an up-to-date value before proceeding.
+      Allows clients to perform consistent read-modify-writes
+      through optimistic concurrency control.
+    output: true
+  - type: NestedObject
+    name: fleet
+    description: Fleet configuration for the cluster.
+    output: true
+    properties:
+      - type: String
+        name: 'membership'
+        description: |
+          The name of the managed Fleet Membership resource associated to this cluster.
+          Membership names are formatted as
+          `projects/<project-number>/locations/<location>/memberships/<cluster-id>`.
+        output: true
+  - type: NestedObject
+    name: status
+    description: ResourceStatus representing detailed cluster state.
+    output: true
+    properties:
+      - type: String
+        name: 'errorMessage'
+        description: |
+          Human-friendly representation of the error message from the admin cluster
+          controller. The error message can be temporary as the admin cluster
+          controller creates a cluster or node pool. If the error message persists
+          for a longer period of time, it can be used to surface error message to
+          indicate real problems requiring user intervention.
+        output: true
+      - type: Array
+        name: 'conditions'
+        description: |
+          ResourceConditions provide a standard mechanism for higher-level status reporting from admin cluster controller.
+        output: true
+        item_type: 
```

```suggestion
        item_type:
```
(remove trailing space)